resources/views/components/quantity.blade.php

```diff
@@ -6,6 +6,7 @@
 <quantity-select v-slot="qtySelect" {{ $attributes }}>
     <div class="flex items-center justify-center border rounded bg-white h-12 self-start">
         <button
+            disabled
```

Comment:
Why have you added this?

Reply:
because the left button is disabled after full pageload, we just set it to disabled before JS loads so you don't have any changes

After JS is loaded it uses that value instead